From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Thu, 29 Jan 2015 22:53:14 -0500
Subject: override-soversions

---
 OpenSceneGraph/src/OpenThreads/CMakeLists.txt | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/OpenSceneGraph/src/OpenThreads/CMakeLists.txt b/OpenSceneGraph/src/OpenThreads/CMakeLists.txt
index 168055e..59ccca8 100644
--- a/OpenSceneGraph/src/OpenThreads/CMakeLists.txt
+++ b/OpenSceneGraph/src/OpenThreads/CMakeLists.txt
@@ -5,7 +5,9 @@ ADD_DEFINITIONS(-DOPENTHREADS_EXPORTS)
 SET(OPENTHREADS_MAJOR_VERSION 2)
 SET(OPENTHREADS_MINOR_VERSION 6)
 SET(OPENTHREADS_PATCH_VERSION 0)
-SET(OPENTHREADS_SOVERSION 12)
+IF(NOT OPENTHREADS_SOVERSION)
+    SET(OPENTHREADS_SOVERSION 12)
+ENDIF (NOT OPENTHREADS_SOVERSION)
 
 SET(OPENTHREADS_VERSION ${OPENTHREADS_MAJOR_VERSION}.${OPENTHREADS_MINOR_VERSION}.${OPENTHREADS_PATCH_VERSION})
 
