From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Thu, 29 Jan 2015 22:53:14 -0500
Subject: Upstream-fix-(b801ae)-for-bug-#736350

Bug-Debian: http://bugs.debian.org/736350
Origin: upstream https://github.com/openscenegraph/osg/commit/b801ae9d499a78889a322b95fbdf9864828349bc
---
 OpenSceneGraph/src/osg/Geometry.cpp | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/OpenSceneGraph/src/osg/Geometry.cpp b/OpenSceneGraph/src/osg/Geometry.cpp
index d758e93..ca0f4ad 100644
--- a/OpenSceneGraph/src/osg/Geometry.cpp
+++ b/OpenSceneGraph/src/osg/Geometry.cpp
@@ -161,12 +161,14 @@ void Geometry::setTexCoordArray(unsigned int index,Array* array, osg::Array::Bin
     if (_texCoordList.size()<=index)
         _texCoordList.resize(index+1);
 
-    if (array && binding!=osg::Array::BIND_UNDEFINED) array->setBinding(binding);
+    if (array)
+    {
+        if (binding!=osg::Array::BIND_UNDEFINED) array->setBinding(binding);
+        else array->setBinding(osg::Array::BIND_PER_VERTEX);
+    }
 
     _texCoordList[index] = array;
 
-    // do we set to array BIND_PER_VERTEX?
-
     dirtyDisplayList();
 
     if (_useVertexBufferObjects && array)
